h3/src/qpack/stream.rs

```diff
@@ -0,0 +1,318 @@
+use bytes::{Buf, BufMut};
+
+use super::{
+    parse_error::ParseError,
+    prefix_int::{self, Error as IntError},
+    prefix_string::{self, Error as StringError},
+};
+
+pub enum EncoderInstruction {
+    InsertWithNameRef,
+    InsertWithoutNameRef,
+    Duplicate,
+    DynamicTableSizeUpdate,
+    Unknown,
+}
+
+impl EncoderInstruction {
+    pub fn decode(first: u8) -> Self {
+        if first & 0b1000_0000 != 0 {
```

Comment:
I'm guessing these constants are shared with the header block decoding bits? Would it make sense to consolidate them into a single set of constants?

Reply:
Actually not, this is another part of the spec.